templates/index.md

```diff
@@ -80,6 +80,10 @@ provider "timescale" {
   secret_key = var.ts_secret_key
 }
 
+provider "aws" {
+  region = "us-east-1"
```

Comment:
Let's use the variable here, to be consistent with the region used in `timescale_peering_connection`.
```suggestion
  region = var.aws_region
```